ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml

```diff
@@ -0,0 +1,32 @@
+base_images:
+  base:
+    name: "4.3"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+images:
+- dockerfile_path: Dockerfile.rhel7
+  from: base
+  to: linuxptp-daemon
```

Comment:
The name of this image should be `ptp-daemon` or just `ptp`.  Will there ever be other images?

Reply:
there are two images in the PR
one is `linuxptp-daemon` which runs as a kube DaemonSet
the other one is for ptp operator, image called `ptp-operator`, which will be a redhat operator that can be installed via redhat operator src.

Reply:
`linuxptp` is one of the implementations of PTP(Precision Time Protocol).
`Linuxptp` rpm package is supported on RHEL.

Reply:
@smarterclayton I can change the image name from `linuxptp-daemon` to `linuxptp` for origin build, shall I also change the OCP image name to the same?

Reply:
Created ticket to change the image name of OCP: https://jira.coreos.com/browse/ART-1205

Reply:
@smarterclayton hi, above ticket has been resolved and marked as Done (OCP image name was changed to `ptp`). `image-references` in `ptp-operator` has been updated to use the right image name `origin-ptp`. I think we should be good to go with one? please let me know if anything else is missing. Thanks!